src/components/forms/formr-part-b/viewSections/ViewSection1.cy-test.tsx

```diff
@@ -0,0 +1,56 @@
+import { mount } from "@cypress/react";
+import { Provider } from "react-redux";
+import { Router } from "react-router-dom";
+import { submittedFormRPartBs } from "../../../../mock-data/submitted-formr-partb";
+import store from "../../../../redux/store/store";
+import ViewSection1 from "./ViewSection1";
+import history from "../../../navigation/history";
+import ViewSectionShouldIncludeThisData, {
+  ISectionDataField
+} from "./ViewSectionTestHelper";
+
+const makeSectionEditButton = (section: number) => {
+  return false;
+};
+
+const formDataToDisplay: ISectionDataField[] = [
+  { fieldName: "forename", format: "" },
+  { fieldName: "surname", format: "" },
+  { fieldName: "gmcNumber", format: "" },
+  { fieldName: "email", format: "" },
+  { fieldName: "localOfficeName", format: "" },
+  { fieldName: "currRevalDate", format: "LocalDate" },
```

Comment:
I tried to get the property types directly from the FormRPartB model interface, but I couldn't get it to work, hence the rather awkward repetition of the 'format' attribute. There might be a better way of tackling this.

Reply:
@ReubenRobertsHEE no quick answer for this one! 🤣

Reply:
Re subset/mapped types, wondering if a combo of Pick and Partial is worth exploring? Or we could always hide everything in a pure cypress/support/commands.js file? Lol